tox.ini

```
@@ -0,0 +1,62 @@
+[tox]
+envlist = py36-tests
```

```suggestion
envlist = tests
```

(See comment above about not duplicating `py36-`), although I think this may be a bug in the cookiecutter

---

This is a cookie cutter thing yes.